api/hasura/cron/_epochs.test.ts

```diff
@@ -643,7 +647,7 @@ describe('epoch Cron Logic', () => {
       const input = getEpochInput('notifyStart', {});
       const result = await notifyEpochStart(input);
       expect(result).toEqual([]);
-      expect(sendSocialMessage).not.toBeCalled();
+      //expect(mockSendSocial).not.toBeCalled();
```

Comment:
just curious why this one is commented out

Reply:
Ah thank you! Will fix